loadtest/control/actions.go

```diff
@@ -729,3 +730,19 @@ func Reload(u user.User) UserActionResponse {
 
 	return UserActionResponse{Info: "page reloaded"}
 }
+
+func CollapsedThreadsEnabled(u user.User) (bool, *UserActionResponse) {
```

Comment:
For consistency we might want to return the value instead/ 
```suggestion
func CollapsedThreadsEnabled(u user.User) (bool, UserActionResponse) {
```